portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns a greeting. */
+@WebServlet("/greeting")
+public class GreetingServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    response.getWriter().println("Hello World!");
+  }
+}
```

Comment:
Nit: Add new line at the end of the document.

Reply:
My bad! Done, thanks!